site/frontend/src/pages/compare/header/filters.vue

```diff
@@ -29,11 +31,13 @@ watch(
   },
   {deep: true}
 );
+
+const opened = createPersistedRef(PREF_FILTER_OPENED);
```

Comment:
The default value could even depend on the filters' value: if there's a filter that has been changed from its default value (via URL) then the panel should probably be open.